gitlab/resource_gitlab_project_level_mr_approvals.go

```diff
@@ -0,0 +1,129 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabProjectLevelMRApprovals() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabProjectLevelMRApprovalsCreate,
+		Read:   resourceGitlabProjectLevelMRApprovalsRead,
+		Update: resourceGitlabProjectLevelMRApprovalsUpdate,
+		Delete: resourceGitlabProjectLevelMRApprovalsDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": {
```

Comment:
I got an error message when I use project name. It would be good to run `acceptance-ee` tests on pull_requests. When we use isRunningInCE in SkipFunc we don't run ee only tests in github actions.

Reply:
Could you change the type to integer then?